test/SFSDKInfoTestSuite.js

```diff
@@ -62,7 +62,7 @@ SDKInfoTestSuite.prototype.testGetInfo = function()  {
             // appName
             QUnit.ok(sdkInfo.appName == "SalesforceHybridSDKTestApp" || sdkInfo.appName == "SalesforceHybridTest", "expected different app name");
             // appVersion
-            QUnit.equal(sdkInfo.appVersion, "1.0", "expected different app version");
+            QUnit.equal(sdkInfo.appVersion, "1.0  (1.0)", "expected different app version");
```

Comment:
On `iOS`, this will be `1.0 (1.0)`. On `Android`, this will probably be `1.0`. We need a conditional here.